symfony/plugins/orangehrmAdminPlugin/test/Dao/LanguageDaoTest.php

```diff
@@ -16,126 +16,110 @@
  * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
  * Boston, MA  02110-1301, USA
  */
-require_once sfConfig::get('sf_test_dir') . '/util/TestDataService.php';
+
+namespace OrangeHRM\Admin\Tests\Dao;
+
+use OrangeHRM\Admin\Dao\LanguageDao;
+use OrangeHRM\Admin\Dto\LanguageSearchFilterParams;
+use OrangeHRM\Config\Config;
+use OrangeHRM\Entity\Language;
+use OrangeHRM\Tests\Util\TestCase;
+use OrangeHRM\Tests\Util\TestDataService;
 
 /**
  * @group Admin
  */
-class LanguageDaoTest extends PHPUnit_Framework_TestCase {
+class LanguageDaoTest extends TestCase
+{
 
-	private $languageDao;
-	protected $fixture;
+    private $languageDao;
+    protected $fixture;
 
-	/**
-	 * Set up method
-	 */
-	protected function setUp() {
-
-		$this->languageDao = new LanguageDao();
-		$this->fixture = sfConfig::get('sf_plugins_dir') . '/orangehrmAdminPlugin/test/fixtures/LanguageDao.yml';
-		TestDataService::populate($this->fixture);
-	}
+    /**
+     * Set up method
+     */
+    protected function setUp(): void
+    {
+        $this->languageDao = new LanguageDao();
+        $this->fixture = Config::get('ohrm_plugins_dir') . '/orangehrmAdminPlugin/test/fixtures/LanguageDao.yml';
```

Comment:
`Config::get('ohrm_plugins_dir')` ==> `Config::get(Config::PLUGINS_DIR)`, find and replace other place as well